bmrg-flow/values.yaml

```diff
@@ -108,6 +108,10 @@ global:
       user: boomerang
       password: passw0rd
       secretName:
+      extraParams:
+      tls:
+        enable: false
+        cert:
```

Comment:
Two questions

1. Can we just point to a secret rather than uploading the cert? I.e. I am guessing when MongoDB installs, it has the crt published in a secret?

2. Can we combine this to be tlsSecretName and just reference that?

Reply:
@tlawrie Whats your thoughts on the above?

Reply:
So a few things here.

1. This section (and the database parameters) are in the global section which is pushed / populated to all sub charts. We want to try and keep this as slim line as possible.

2. What is the contents of the cert? Its a TLS certificate yeah? 

3. If the MongoDB is installed in cluster, does it generate the cert and stick it in the same secret as the base64 encoded username and password? Or do you still have to manually upload it?

4. If installed outside cluster, what details do you need? is it just TLS cert or do you still need username and password?